lib/data_pull.rb

```diff
@@ -197,6 +200,61 @@ def run(args:, config:)
     end
   end
 
+  class SsnSignatureReport
+    def run(args:, config:)
+      require 'data_requests/deployed'
+      ssns = args
+
+      normalized_ssns = ssns.map { |ssn| SsnFormatter.normalize(ssn) }
```

Comment:
It doesn't look like we normalize the SSN on creating the profile would this cause an issue?

Reply:
Haven't been able to find out where we normalize the SSN on creating the profile. The closest is where we validate the SSN - https://github.com/18F/identity-idp/blob/9a73f763f56344d614217bac88865a4c92d4324b/app/validators/idv/form_ssn_format_validator.rb#L9.

Reply:
Scratch that. That regex would not normalize anything as it would pass both "123456789" and "123-45-6789". @mdiarra3 has found a bug where we don't normalize, at least in the local environments. I have reproduced it in dev. Found it hard to reproduce in cloud deployments as https://developers.login.gov/testing/#testing-identity-proofing doesn't seem to be entirely correct.

Reply:
Created a PR to fix this - https://github.com/18F/identity-idp/pull/12091.